requirements/dev

```diff
@@ -1,7 +1,7 @@
 -r prod
-pytest
-pytest-cov
-pytest-tornado
-pytest-flakes
-coverage
-requests
+pytest==2.7.2
```

Comment:
It is not needed as dev pulls in all prod requirements via `-r prod`. Or do you mean that we should pull in all dev requirements for `arteria-core`as well?